.github/ISSUE_TEMPLATE/New_feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: New Feature Request
+about: Request for engineering work on a new feature
+title: 'New Feature Request'
+labels: 
+assignees: '@abbylindsay'
+
+---
+
+**Describe the work you would like completed:**
+<!-- A clear and concise description of what problem you are trying to solve -->
+**Is this request related to a problem:**
+<!-- Is this simply a new feature or is this to resolve a different issue -->
+**Describe alternatives you have considered:**
+<!-- Does this work need to be done? Is there a simpler solution? -->
+**Priority level:**
+<!-- 1- a V&V will not be run for this issue alone. 2- Want to have 3- Causing minor impact to study/participant 4- Causing big impact to study/participant 5- BAD BAD BAD -->
+**New UI or UI Update? Screenshot(s):**
+<!-- Please be specific what the UI should be - size, color, shape, location, etc. -->
+**Additional Context:**
+<!-- The more the better. -->
```

Comment:
It may help to capture what component(s) this will touch (if known). Operations may not know so it may be something we determine.